File: 8 Kyu/Sum of positive/positiveSum.js
// Kata:
// https://www.codewars.com/kata/5715eaedb436cf5606000381

// Example:

// Solution:
const positiveSum = (arr) => {
  const sum = arr.filter(el => el > 0);
  return sum.length === 0 ? 0 : sum.reduce( (a,b) => a + b);
}

File: 8 Kyu/Sum of positive/positive_sum.rb
# Kata:
# https://www.codewars.com/kata/5715eaedb436cf5606000381

# Example:

# Solution:
def positive_sum(arr)
  arr.select! { |num| num > 0 }
  arr.empty? ? 0 : arr.reduce(0, :+)
end